app/views/home/_event_card.html.erb

```diff
@@ -0,0 +1,23 @@
+<% @events.each do |event| %>
+<div class="text-center shadow-lg rounded bg-light w-25 py-2 mb-2 mr-2">
+<h4 class="text-info font-weight-bold">TITLE</h4>
+<span class="font-weight-normal text-dark"><%= event.title %></span>
+<h4 class="text-info font-weight-bold">description</h4>
+<p><%= event.description %></p>
+<h4 class="text-info font-weight-bold">date</h4>
+<p><%= event.time %></p>
+<div class="w-100 bg-secondary text-light py-2 mt-3 font-weight-bold"><%= link_to 'See Attendees', event %></div>
+<div class="w-100 bg-secondary text-light py-2 mt-3 font-weight-bold">Created by: <%= User.find(event.creator_id).username %></div>
```

Comment:
- [ ] Use a helper method to get this User.find(event.creator_id).username instead of calling the model directly in the view